### ISSUE TITLE
Glove80 (nRF52840 chipset) fails to connect to a Sounix Bluetooth 5.3 adapter

### ISSUE THREAD

**Issue by Intuos5** [zmkfirmware/zmk#2061]
I recently bought a Sounix 5.3 Bluetooth adapter to connect my Glove80 with. In Windows 10, I am unable to connect to the Bluetooth adapter. Stephen from MoErgo advised me to create this issue.

The Glove80 uses a nRF52840 chipset for its Bluetooth connection. Windows downloads a Realtek 5.3 driver (version 1.9.1038.3008) for the Sounix Bluetooth adapter, so I assume the adapter uses a Realtek chip.

**Comment by caksoylar** [comment]
This is probably covered by the tweaks and testing that is currently going on in ZMK and MoErgo Discords. Curren recommendation is to use try the `CONFIG_ZMK_BLE_EXPERIMENTAL_CONN=y` flag documented in https://zmk.dev/docs/config/bluetooth.

**Comment by madushan1000** [comment]
`CONFIG_ZMK_BLE_EXPERIMENTAL_CONN=y` is working okay so far with RTL8852AE.

**Comment by saladbowlwithdressing** [comment]
> `CONFIG_ZMK_BLE_EXPERIMENTAL_CONN=y` is working okay so far with RTL8852AE on linux 6.5.0.

where can i find this setting? i am looking at the layout editor and i cant find it anywhere.

**Comment by madushan1000** [comment]
you have to put it in a file like this https://github.com/madushan1000/taira-keyboard-zmk-config/blob/main/config/taira.conf#L4

**Comment by saladbowlwithdressing** [comment]
> you have to put it in a file like this https://github.com/madushan1000/taira-keyboard-zmk-config/blob/main/config/taira.conf#L4

pardon my ignorance, I have zero programming skills and am completely new to this, so all of this is still quite confusing. I use the glove80, do I just drag a file with that text inside both halves? I appreciate the help!

**Comment by madushan1000** [comment]
no, you need a custom firmware, I guess you would have to ask the makers of glove80 for one.

**Comment by saladbowlwithdressing** [comment]
> no, you need a custom firmware, I guess you would have to ask the makers of glove80 for one.

ahh that's too bad then, thank you regardless!

**Comment by caksoylar** [comment]
@saladbowlwithdressing Please see the 2nd/3rd items in https://www.moergo.com/pages/troubleshooting-faq. MoErgo bakes in these settings with their new firmware versions, which should ideally resolve your issue.
